.eslintrc.json

```diff
@@ -127,6 +127,14 @@
     "import/resolver": {
       "alias": {
         "map": [
+          [
+            "@/(.*\\.svg)\\?url",
+            "assets/"
+          ],
+          [
+            "@/(.*)",
+            "assets/"
+          ],
```

Comment:
Is the first element (with added `?url`) needed? Is it not matched by the second? I would expect that to match everything 🤔

Reply:
As discussed, this appears to be required because otherwise, the reported ESLint error is raised. Thanks!